src/patterns/ec2-app/base.ts

```diff
@@ -347,6 +349,8 @@ export class GuEc2App extends Construct {
       userData,
       withoutImdsv2,
       imageRecipe,
+      vpcOverride,
```

Comment:
I think using destructuring with default values here would make it more obvious what the default is, and removes the `??` below. For example:

```ts
const { vpc = THE_DEFAULT } = props;
```

Reply:
👍